src/todos/create-todo.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { ETodoPriority } from 'src/common/enums/todo-priority.enum';
+import { IsDateString, IsEnum, IsNotEmpty, IsString } from 'class-validator';
+
+export class CreateTodoDto {
+  @IsNotEmpty()
+  @IsString()
+  title: string;
```

Comment:
```suggestion
  title!: string;
```

We need to let typescript know that this will always be assigned a value.